frontend/src/components/buttons/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import "./Button.css";
+
+interface iButton {
+    onClick? ():any;
+    width?: string;
+    height?: string;
+    bcolor? : string;
+    color? : string;
+    text?: string;
+    navbar?: boolean;
+    toolbar?: boolean;
+    menu?: boolean;
+    children?: string;
```

Comment:
For default props, include it in `defaultProps` as best practice. Refer to [BackCardContents.tsx](https://github.com/devalparikh/BusiCard/blob/master/frontend/src/components/cards/BackCardContents.tsx)